.circleci/config.yml

```diff
@@ -245,7 +245,7 @@ jobs:
             go test
   evidence_rails_build:
     working_directory: ~/Empirical-Core
-    parallelism: 4
+    parallelism: 1
```

Comment:
Can you drop a note in the main PR description expanding on why we want to do this?  I don't object or anything, as it's a pretty small test suite, so I doubt it will have much real world impact on our development cycles, but I'd like to know why.

Reply:
@anathomical Sure thing, I've added a screenshot for more clarity.